vllm/entrypoints/openai/rpc/client.py

```diff
@@ -85,14 +88,19 @@ async def _send_get_data_rpc_request(self, request: RPCUtilityRequest,
 
         return data
 
-    async def _send_one_way_rpc_request(self, request: RPC_REQUEST_TYPE,
-                                        error_message: str):
+    async def _send_one_way_rpc_request(self,
+                                        request: RPC_REQUEST_TYPE,
+                                        error_message: str,
+                                        timeout: int = 0):
```

Comment:
I think default here should be `None` rather than 0. Could you include `Optional[int]` type

Reply:
Oh yeah, C++ programmer habits...